src/coreclr/src/inc/readytorun.h

```diff
@@ -395,4 +395,13 @@ enum ReadyToRunRuntimeConstants : DWORD
     READYTORUN_ReversePInvokeTransitionFrameSizeInPointerUnits = 2
 };
 
+enum ReadyToRunHFAElemType : DWORD
+{
+    READYTORUN_HFA_ELEMTYPE_None = 0,
+    READYTORUN_HFA_ELEMTYPE_Float = 1,
+    READYTORUN_HFA_ELEMTYPE_Double = 2,
```

Comment:
Note that `float`/`double` naming is specific to C# (and C/C++).  For instance, in F# `float` is a [synonym](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/basic-types) for `double`.  VB .NET has the `Double` keyword, but no `Float` one. For that reason I would prefer either `Float32`/`Float64` as in the CIL assembler or `Single`/`Double` as in the corresponding .NET type names.